thunder.install

```diff
@@ -311,9 +312,51 @@ function thunder_update_8102() {
  */
 function thunder_update_8103() {
 
-  \Drupal::service('module_installer')->install([
-    'thunder_updater',
-  ]);
+  /** @var \Drupal\thunder\ThunderUpdateLogger $updateLogger */
+  $updateLogger = Drupal::service('logger.thunder_update');
+  $updateLogger->cleanLogs();
+
+  $moduleInstaller = \Drupal::service('module_installer');
+
+  try {
+    if (!$moduleInstaller->install(['thunder_updater'])) {
+      $updateLogger->warning(t('Unable to enable the Thunder Updater'));
+    }
+  }
+  catch (MissingDependencyException $e) {
+    $updateLogger->warning(t('Unable to enable the Thunder Updater because of missing dependencies.'));
+  }
 
   \Drupal::service('thunder_updater')->checkAllListPoints(FALSE);
```

Comment:
Thunder updater service should be assigned to variable -> and then used on multiple places.